src/fsql/query.py

```diff
@@ -83,6 +83,13 @@ def f(**kwargs):  # TODO how to define f(`column`) ? That would simplify AtomicQ
     return AtomicQuery(f, set([column]))
 
 
+def Q_IN(column: str, values: Iterable):
+    def f(**columns):
```

Comment:
I would prefer to be consistent and rather use `**kwargs` instead of `**columns` as it's done in `Q_EQ`.

Reply:
I could rename it back to `**kwargs`, but IMO this is much more readable, because the content of this parameter becomes apparent. Another alternative is to refactor this to `f(values: dict)` instead of `f(**values)`, in `**columns` is aesthetically unacceptable. What do you think?

Reply:
I am not much into aesthetics 😄 but it's good to be consistent. Agree that `**kwargs` is not the best option. On the other hand `columns` might be misleading as well (what's is the content? I do not think it's so apparent), but since it's already used in `AtomicQuery` and other places, you are right it's probably better to use it (and agree that more readable then `kwargs`). Just ensure it's used in `Q_EQ` as well.